### ISSUE TITLE
🐛 Allow easy picture edition on tag

### ISSUE THREAD

**Issue by Tirodem** [be-BOP-io-SA/be-BOP#832]
(for early 2024)
Currently, once a tag is created, you cannot edit the tag pictures

**Comment by ithiame** [comment]
![image](https://github.com/user-attachments/assets/bc1422a8-5c43-4e3f-83c5-0dd63f34379f)

On tag update we can delete picture and add missing pictures.

**Comment by Tirodem** [comment]
Indeed, we fixed this since on another ticket. We can close this one.
@AmadouAgiltoo just please check this on main and close it.
